vfcompositor/Compositor.c

```
@@ -0,0 +1,784 @@
+                    g_window_list_tail->next = top_window;
+                g_window_list_tail = top_window;
+            }
+            // Check if the click is on the title bar
+            if (y - top_window->rect.y < 20) {
+                g_focused_window->is_moving = true;
+            }
+        }
+    }
+}
+
+void OnMouseButtonUp(int x, int y, uint8_t button) {
```

Clang (14.0.6)

[warning] 3 adjacent parameters of 'OnMouseButtonUp' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)